c++/test/TestPredicatePushdown.cc

```diff
@@ -281,6 +282,45 @@ namespace orc {
     for (uint64_t i = 0; i < 2000; ++i) {
       EXPECT_EQ(i + 3500 , batch1.data[i]);
     }
+    // 2nd batch of 1500 rows
```

Comment:
Please make a new test case, @stiga-huang .

Reply:
This line belongs to the current test. I'll move the following test of "Sargs: col1 < 3500" to a separated test.

Reply:
+1 for that. Thanks!